examples/src/main/java/com/squareup/moshi/recipes/ReadJsonList.java

```diff
@@ -1,3 +1,5 @@
+package com.squareup.moshi.recipes;
```

Comment:
Please move this back to its original spot

Reply:
Done.